ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -76,14 +76,8 @@ enum CommandDeduplicationType {
 
 // See `daml-lf/spec/contract-id.rst` for more information on contract ID formats.
 message ExperimentalContractIds {
-  ContractIdV0Support v0 = 1;
```

Comment:
I suggest adding `reserved 1`.

Alternatively, you can renumber `v1` as we haven't released this yet.

Reply:
renumbered, doesn’t seem useful to keep it around.